src/components/shared/map/MapCenter.tsx

```diff
@@ -57,27 +58,27 @@ const MapCenter = () => {
             setCenter({ lat: 50.064192, lng: 19.944544 });
           }
         } catch (error) {
-          logger.error('Error fetching map center:', { error });
```

Comment:
It is generally recommended to pass the error as an object `({ error })` to promote structured logging.  Furthermore, the Sentry logging tool can better parse and index more effectively which allows for better troubleshooting when issues occur.